images/facebook.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 42 (36781) - http://www.bohemiancoding.com/sketch -->
+    <title>BD39F898-CC24-4E34-BEB5-9EB83DE73341</title>
+    <desc>Created with sketchtool.</desc>
+    <defs>
+        <linearGradient x1="0%" y1="0%" x2="100%" y2="100%" id="linearGradient-1">
+            <stop stop-color="#FD7D57" offset="0%"></stop>
+            <stop stop-color="#FA2471" offset="100%"></stop>
+        </linearGradient>
+    </defs>
+    <g id="1st-Version" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="1st-version-mobile" transform="translate(-13.000000, -511.000000)" fill="url(#linearGradient-1)">
+            <path d="M30.00625,511 L13.99375,511 C13.44475,511 13,511.44475 13,511.99375 L13,528.007 C13,528.55525 13.44475,529 13.99375,529 L22.615,529 L22.615,522.0295 L20.269,522.0295 L20.269,519.313 L22.615,519.313 L22.615,517.30975 C22.615,514.98475 24.03475,513.71875 26.10925,513.71875 C27.103,513.71875 27.9565,513.793 28.2055,513.826 L28.2055,516.256 L26.767,516.25675 C25.639,516.25675 25.42075,516.793 25.42075,517.579 L25.42075,519.31375 L28.111,519.31375 L27.76075,522.03025 L25.42075,522.03025 L25.42075,529 L30.00775,529 C30.55525,529 31,528.55525 31,528.00625 L31,511.99375 C31,511.44475 30.55525,511 30.00625,511 L30.00625,511 Z" id="Shape"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
can you clean this up with `svgo`?

Or even better, is this something that can be integrated with gatsby? https://github.com/rpominov/svgo-loader

PS: I have a standard svgo.yml file that I use for all projects (and I should actually submit it to our guides). Let me know if you need it